unity/Assets/Maroon/scenes/experiments/3DMotionSimulation/Scripts/SkyboxController.cs

```diff
@@ -76,6 +76,5 @@ public static SkyboxController Instance
     /// </summary>
     public void ResetObject()
     {
-        SetExperimentRoom();
     }
```

Comment:
If this empty method is unused, and you also removed IResetObject inheritance, then I think we can remove the whole method.